Control flow exercises

danpottshimself opened the pull request:
All tasks have been completed for control flow slides.

dpotts7 committed:
3/5 tasks completed.

Triangle created using astrix.
Equilateral triangle created in a similar method.
Fibonacci sequence created using JS.

dpotts7 committed:
Added task 4/5.

Loop through numbers 1-1­00:
-If the number is divisible by 3, output Java
-If the number is divisible by 5, output Script
-If the number is divisible by both 3 and 5, output JavaScript
-Otherwise output the number.

dpotts7 committed:
started arrays

dpotts7 committed:
All tasks completed.

Control flow exercises.

keithbarrow commented on control-flow-exercises.js:
```diff
@@ -0,0 +1,65 @@
+ */
+//Create a triangle using astrix.
+var numLines = 2;
+var stars = ("*");
```
First: String literals should be '_' not "_"
Second: No brackets - these achieve anything other than making it look like a TIE fighter.

keithbarrow commented on control-flow-exercises.js:
```diff
@@ -0,0 +1,65 @@
+console.log(anArray);
+
+//Loop of 100 stating Java or Script when divisible by 3 or 5.
+for (var k =0; k<100; k++){
```
You should re-look at this, several deficiencies:
1. !=, vs !==
2. Work out whether it is a 'JavaScript' output first, then else-if for 'Java' then else for 'Script'

This is a well known problem in IT - search for "FizzBuzz Problem". There isn't a best solution, except depending upon your perspective.

dpotts7 committed:
changed the order of the java script phrases and also added an else statement which displays the number.

I added this because my console previously read java, script and javascript as well as the number.

keithbarrow commented:
:shipit:
